helpers/bajasHelper/bajasHelper.helper.go

```diff
@@ -272,7 +319,7 @@ func GetAllSolicitudes() (historicoActa []map[string]interface{}, outputError ma
 	}
 }
 
-func TraerDetalle(id int) (Solicitud map[string]interface{}, outputError map[string]interface{}) {
+func TraerDetalle(id int) (Baja *models.TrBaja, outputError map[string]interface{}) {
```

Comment:
Falta documentar, cambiar `...` en el _suggested change_, digamos "TraerDetalle hace tal cosa"
```suggestion
// TraerDetalle ...
func TraerDetalle(id int) (Baja *models.TrBaja, outputError map[string]interface{}) {
```